libs/axios.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+export const axiosClient = axios.create({
+  baseURL: process.env.NEXT_PUBLIC_API,
```

Comment:
I tried adding the .env files but it wasn't working for me, I could have it in the wrong directory tho. Asside from this it's good to be merged